### PR TITLE
Update BSPs

### PR DESCRIPTION
**Pull request by nategraff-sifive**
Update using the latest freedom-devicetree-tools.

### CODE REVIEW

**Comment by sifivekevin** [comment]
LGTM,  These will find there way into v201905-rc2 (and final)?

**Comment by nategraff-sifive** [comment]
Yep, the goal is to put them in the next RC

File: bsp/sifive-hifive-unleashed/openocd.cfg [hunk 1]
```diff
@@ -1,24 +1,30 @@
+#write_config_file
+# JTAG adapter setup
 adapter_khz     10000
 
 interface ftdi
-ftdi_device_desc "Dual RS232-HS"
-ftdi_vid_pid 0x0403 0x6010
+ftdi_device_desc "Olimex OpenOCD JTAG ARM-USB-TINY-H"
```
Comment (cgsfv) [inline bsp/sifive-hifive-unleashed/openocd.cfg]: The HiFive Unleashed does not use Olimex, it uses FTDI, right?

File: bsp/sifive-hifive1/openocd.cfg [hunk 2]
```diff
@@ -1,34 +1,30 @@
+#write_config_file
+# JTAG adapter setup
 adapter_khz     10000
 
 interface ftdi
-ftdi_device_desc "Dual RS232-HS"
-ftdi_vid_pid 0x0403 0x6010
+ftdi_device_desc "Olimex OpenOCD JTAG ARM-USB-TINY-H"
```
Comment (cgsfv) [inline bsp/sifive-hifive1/openocd.cfg]: Same here, shouldn't it also be FTDI (Dual RS232-HS)?

**Comment by cgsfv** [comment]
But besides that it looks good

**Comment by nategraff-sifive** [comment]
Thanks for the catch, Carsten, we're generating openocd.cfg now so I need to update the script to fix that.

**Pull request merged**